WcaOnRails/app/models/format.rb

```diff
@@ -8,4 +8,16 @@ class Format < ApplicationRecord
   has_many :events, through: :preferred_formats
 
   scope :recommended, -> { where("ranking = 1") }
+
+  def serializable_hash(options = nil)
+    {
+      id: self.id,
+      name: self.name,
+      sort_by: self.sort_by,
+      sort_by_second: self.sort_by_second,
+      expected_solve_count: self.expected_solve_count,
+      trim_fastest_n: self.trim_fastest_n,
+      trim_slowest_n: self.trim_slowest_n,
+    }
```

Comment:
@viroulep actually had a different suggestion to handle this using ActiveSupport concerns. I'm supportive of doing that, but we're going to addresss that in a later PR.